Can't build documentation locally

Using the same environment as in CI (py 3.7, sphinx 3.5.4), where things compile fine, I get (on a fresh build):
```
$ make all
sphinx-build -b html -d build/doctrees  source -W build/html
Running Sphinx v3.5.4
making output directory... done
creating gallery file entries... [100%] logaxis.rst                                                                                          
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://pandas.pydata.org/pandas-docs/stable/objects.inv...
loading intersphinx inventory from https://numpy.org/doc/stable/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 245 source files that are out of date
updating environment: [new config] 245 added, 0 changed, 0 removed
copying bokeh-plot files... [100%] bokeh-plot-f583dd6b75c44e6784c4bbc5043a714f-external-docs-first_steps-first_steps_2.js                    

Exception occurred:
  File "/home/mateusz/conda/envs/bk_py37/lib/python3.7/site-packages/docutils/parsers/rst/states.py", line 882, in interpreted
    nodes[0][0].rawsource = unescape(text, True)
AttributeError: 'str' object has no attribute 'rawsource'
The full traceback has been saved in /tmp/sphinx-err-ny5pt50k.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:21: html] Error 2
```
@bokeh/dev, did anyone encounter this before? I have a suspicion this is related to one of our extensions, and I would like to rule that out, if I have submit a bug report upstream.

--- a/sphinx/source/docs/dev_guide/setup.rst
+++ b/sphinx/source/docs/dev_guide/setup.rst
@@ -82,7 +82,7 @@ At the top level of the repository, issue the following command in a terminal:
 
 .. code-block:: sh
 
-    conda env create environment.yml
+    conda env create -f environment.yml
 
 Then, activate the environment:
 

